### PR TITLE
Np tour

### PR DESCRIPTION
**Pull request by nathanpabst**
testing criteria: tour schedule should be visible with a clickable button for tickets.

### CODE REVIEW

File: javascript/main.js [hunk 1]
```diff
@@ -241,7 +242,120 @@ function buildSongList(songArray){
...
+      state: "California",
+      time: "7:00 PM",
+      venue: "LA Forum",
+      tickets: "suckit.html"
+  },
+];
+
+function createEvent(myEvent){
+  var tourString = "";
+  console.log("hi");
```
Comment (amillion3) [inline javascript/main.js]: remove console.log

File: javascript/main.js [hunk 2]
```diff
@@ -241,7 +242,120 @@ function buildSongList(songArray){
...
+    tourString += '<article class="events">';
+    tourString += '<p class="date">' + myEvent[i].date + '</p>';
+    
+    tourString += '<p class="city">' + myEvent[i].city + '</p>';
+    tourString += '<p class="state">' + myEvent[i].state + '</p>';
+    tourString += '<p class="time">' + myEvent[i].time + '</p>';
+    tourString += '<p class="venue">' + myEvent[i].venue + '</p>';
+    tourString += '<a href="' + myEvent[i].tickets + '"><button>Buy Now</button></a>';
+    tourString += '</article>';
+    console.log(tourString);
```
Comment (amillion3) [inline javascript/main.js]: remove console.log

**Comment by aprilrochelle** [comment]
Widen columns

**Comment by aprilrochelle** [comment]
Add H1 tag to html

**Comment by aprilrochelle** [comment]
👍 Looks good to me

File: javascript/main.js (hunk 2 above)
Comment (amillion3) [reply javascript/main.js]: A second console.log at line 350

**Comment by amillion3** [comment]
Looks good 👍

**Comment by ASlayton** [comment]
👍

**Pull request merged**